app/model/account_transaction_details.py

```diff
@@ -1,18 +1,51 @@
 from app import db
-from datetime import datetime
-from app.model.bank_account import bank_account
 
-class account_transaction_details(db.Model):
+
+class AccountTransactionDetails(db.Model):
+
+    """Create table AccountTransactionDetails in the database"""
+
+    __tablename__ = 'AccountTransactionDetails'
     id = db.Column(db.Integer, primary_key=True)
     transaction_amount = db.Column(db.Integer)
-    transaction_date = db.Column(db.Date)
-    account_number = db.Column(db.Integer, db.Foreignkey('account_number'))
-    transaction_type_id = db.Column(db.Integer, db.Foreignkey('transaction_type_id'))
-    fund_id = db.Column(db.Integer, db.Foreignkey('fund_id'))
+    transaction_date = db.Column(db.DateTime, server_default=db.func.now())
+    bank_account_id = db.Column(db.Integer, db.ForeignKey('BankAccount.id'))
+    transaction_type_id = db.Column(db.Integer, db.ForeignKey('TransactionType.id'))
+    fund_id = db.Column(db.Integer, db.ForeignKey('FundTransfer.id'))
```

Comment:
This should be only `id` as this is `id` for `transaction`. and change it at all places required.

Reply:
have taken the id for account transaction details while others are foreign keys.